scanner/src/main/kotlin/CloudStorageCache.kt

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2017-2018 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.scanner
+
+import ch.frankel.slf4k.*
+import com.google.auth.oauth2.ServiceAccountCredentials
+import com.google.cloud.storage.Storage
+import com.google.cloud.storage.StorageOptions
+import com.here.ort.model.*
```

Comment:
Nit: Please do not use wildcard import here. We've allowed wildcard imports only for a few packages, see

    https://github.com/heremaps/oss-review-toolkit/blob/master/detekt.yml#L38

Also, in the special case of  `com.` imports, we also separate imports from different `com.` packages by blank lines.